powerauth-java-server/src/main/resources/application.properties

```diff
@@ -107,4 +107,6 @@ powerauth.service.correlation-header.enabled=false
 powerauth.service.correlation-header.name=X-Correlation-ID
 powerauth.service.correlation-header.value.validation-regexp=[a-zA-Z0-9\\-]{8,1024}
 # For logging correlation HTTP headers enable the pattern and update correlation header name in the pattern
-#logging.pattern.console=%clr(%d{${LOG_DATEFORMAT_PATTERN:yyyy-MM-dd HH:mm:ss.SSS}}){faint} %clr(${LOG_LEVEL_PATTERN:%5p}) [%X{X-Correlation-ID}] %clr(%5p) %clr(${PID: }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}
\ No newline at end of file
+#logging.pattern.console=%clr(%d{${LOG_DATEFORMAT_PATTERN:yyyy-MM-dd HH:mm:ss.SSS}}){faint} %clr(${LOG_LEVEL_PATTERN:%5p}) [%X{X-Correlation-ID}] %clr(%5p) %clr(${PID: }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n${LOG_EXCEPTION_CONVERSION_WORD:%wEx}
+
+powerauth.service.proximity-check.otp.length=6
```

Comment:
Can we use a longer OTP for proximity check by default?

Reply:
Ok, let's increase it to 8.